gameMechanics.js

```diff
@@ -0,0 +1,303 @@
+// Game Mechanics for the Connect Four Game
+
+// Main methods
+
+var createGrid = function(height, width) {
+	var grid = new Array(width);
+	for (column=0; column < width; column++) {
+		grid[column]=new Array(height);
+	}
+	for (column=0; column < width; column++) {
+		for (row=0; row < height; row++) {
+			grid[column][row] = "<font color='gray'>E</font>";
```

Comment:
Hm... food for thought: Injecting HTML into the grid works, but what if in the future, you decide you want to render your game not using HTML? Or what if you decide to render it using HTML but with neon pink and slime green player pieces?